Linphone/Core/CoreHttpClient.cs

```diff
@@ -1,21 +1,19 @@
 ﻿using BelledonneCommunications.Linphone.Commons;
 using BelledonneCommunications.Linphone.Presentation.Dto;
 using BSN.Resa.Vns.Commons.Responses;
+using Microsoft.Extensions.Options;
 using Serilog;
 using System;
 using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace BelledonneCommunications.Linphone.Core
 {
-    internal class CoreHttpClient
+    public class CoreHttpClient: ICoreHttpClient
```

Comment:
Please move all this code to the data leyer, like Mobayyen's hardware project architecture (please see directory structure and layering model)

Reply:
Recall to what we done on AxB project, where all httpClients were defined in service layer. I just preferred to keep it there. but i will move on ur order.

Reply:
Please move and separate this in data layer such as Mobayyen Hardware Project

Reply:
Resolved.